other/copy-namespace-labels/.chainsaw-test/resource.yaml

```diff
@@ -0,0 +1,85 @@
+#Deployment with Owner Label and additional labels
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-deployment-1
+  namespace: within-ns
+  labels:
+    app: good-app    
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: good-app
+  template:
+    metadata:
+      labels:
+        app: good-app
+    spec:
+      containers:
+      - name: good-app-deploy
+        image: busybox:1.35
+---
+#Deployment with `kubernetes.io/metadata.name` label
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-deployment-2
+  namespace: within-ns
+  labels:
+    kubernetes.io/metadata.name: "my-deployment-2"
```

Comment:
This shouldn't be here. Please remove. The label `kubernetes.io/metadata.name` is only found at the Namespace level.